src/Symfony/Component/Notifier/Bridge/GoogleChat/Tests/GoogleChatTransportFactoryTest.php

```diff
@@ -47,6 +47,7 @@ public function supportsProvider(): iterable
     public function incompleteDsnProvider(): iterable
     {
         yield 'missing credentials' => ['googlechat://chat.googleapis.com/v1/spaces/AAAAA_YYYYY/messages'];
```

Comment:
I think this test is wrong, as the out DSN should not contains `v1/spaces` the exception might not be related to the missing credentials

Reply:
This should then be fixed in 5.2 branch, but I think you are right 🤔

Reply:
Created a PR: https://github.com/symfony/symfony/pull/39848